lambdas/service/app.py

```diff
@@ -600,8 +600,8 @@ def custom_health(keys: Optional[str] = None):
     return app.health_controller.custom_health(keys)
 
 
-@app.schedule('rate(1 minute)', name=config.service_cache_health_lambda_basename)
-def update_health_cache(_event: chalice.app.CloudWatchEvent):
+@app.schedule('rate(1 minute)')
+def servicecachehealth(_event: chalice.app.CloudWatchEvent):
```

Comment:
```suggestion
@app.schedule('rate(1 minute)', name='servicecachehealth')
def update_health_cache(_event: chalice.app.CloudWatchEvent):
```

would avoid the ugly Python function name and …